samtools.1

```diff
@@ -1202,6 +1202,9 @@ regarding check results to be printed to stderr.
 .B -q
 Quiet mode: disables warning messages on stderr about files that fail.
 If both -q and -v options are used then the appropriate level of -v takes precedence.
+.TP
+.B -u
+Expect unmapped input data, so do require targets in the header.
```

Comment:
Did you mean do **not** require targets here?

Reply:
Yep. Pushing a correction.